lib/operations/misc/indexOf.js

```diff
@@ -0,0 +1,7 @@
+function indexOf(a, b) {
+  if (!b || typeof b.indexOf === 'undefined') return false;
+  return b.indexOf(a) !== -1;
+}
+
+indexOf.code = 'in';
```

Comment:
Same here `code` as described in https://github.com/axa-ch/json-logic-js/pull/1/files#r259730014

`in` is an existing operator